dataprofiler/data_readers/data_utils.py

```diff
@@ -180,7 +204,8 @@ def read_json_df(data_generator, selected_columns=None, read_in_string=False):
                 ),
                 ignore_dicts=True,
             )
-            lines.append(obj)
+            if isinstance(obj, dict):  # shold always pass but needed for mypy
+                lines.append(obj)
```

Comment:
based on `unicode_to_str` could be `str`, `list`, or `dict`?

Reply:
json.loads with our object_hook set to dict makes it so unicode_to_str gets a dict (and returns a dict). I don't think mypy can detect that unless we explicitly tell it that it will be always be a dict.

Reply:
Hmm, my concern is a silent error could happen if somehow data returned that wasn't a dict. We'd lose our on that information unless our intent was to purposefully skip anything that wasn't a dict. This might be a case where casting is better and we can handle a real fix later.